lib/vanagon/platform/windows.rb

```diff
@@ -103,7 +104,7 @@ def generate_nuget_package(project)
       # @return [Array] list of commands required to build an msi package for the given project from a tarball
       def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
-        cg_name = "compfiles"
+        cg_name = "ProductComponentGroup"
```

Comment:
If we're only using it in two places, and it doesn't matter if multiple msis installed on the same system have this set the same, then I vote we keep it hard coded.